eng/dockerfile-templates/sdk/Dockerfile.install-reference-assemblies

```diff
@@ -0,0 +1,24 @@
+{{
+    set wrap(s) to:{{return cat("'", s, "'")}} ^
+
+    set versions to ["net40", "net45", "net451", "net452", "net46", "net461", "net462", "net47", "net471", "net472", "net48", "net481"] ^
+    set versions to map(versions, wrap) ^
+
+    set packageVersion to VARIABLES["referenceassemblies|version"] ^
+    set packageInstallDir to "packages"
+
+}}# Install Targeting Packs
+RUN powershell -Command "`
+    New-Item -ItemType Directory -Path \"${Env:ProgramFiles(x86)}\Reference Assemblies\Microsoft\Framework\.NETFramework\"; `
```

Comment:
I think this should be `${Env:ProgramFiles(x86)}\Reference Assemblies\Microsoft\Framework\` , and it could also be a variable as it is reused in `Copy-Item`

Reply:
Fixed in https://github.com/microsoft/dotnet-framework-docker/pull/1224/commits/1b56cd85e84d394a5102703120f33fd6c9f19176